src/models/transfers.js

```diff
@@ -434,7 +434,10 @@ function * setTransfer (transfer, requestingUser) {
   }
 }
 
-function * getFulfillment (transferId) {
+function * getFulfillment (transferId, requestingUser) {
+  if (!requestingUser || !requestingUser.name) {
+    throw new UnauthorizedError('Not authorized')
```

Comment:
Do we want to check that the user is authorized to view the transfer that this fulfillment corresponds to?